draft-bradleylundberg-cfrg-arkg.md

```diff
@@ -379,6 +379,22 @@ ARKG-Generate-Seed() -> (pk, sk)
     sk = (sk_kem, sk_bl)
 ~~~
 
+### Deterministic key generation
+
+Although the above definition expresses the key generation as opaque,
+likely sampling random key distributions,
+implementations MAY choose to implement the functions `BL-Generate-Keypair()`,
+`KEM-Generate-Keypair()` and `ARKG-Generate-Seed()`
+as deriving keys deterministically from some given input key material.
+This can be thought of as defining a single-use ARKG instance where these functions return a constant result.
```

Comment:
"This can be thought of as defining a single-use ARKG instance where these function outputs are static."